### PR TITLE
Reed-Me

### PR DESCRIPTION
**Pull request by rahimialijan**
@dieum512, @Tshobohwa , @binodbhusal please check the content of the readme file and your profile links, if there is any issue let me know

# Add Read-Me
In this pull request, I added a descriptive read-me file and fixed the linter error

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,201 @@
...
+<!-- ER diagram"-->
 
-* Deployment instructions
+# Database Schema <a name="ER-diagram"></a>
 
-* ...
+<div align="center">
+ <img src="https://github.com/binodbhusal/final_capstone_backend/blob/dev/app/ER_diagram/drawSQL-motorhub-final-capstone-project-export-2023-09-29.png" alt="schema" width="600"  height="auto" />
+  <br/>
+</div>
```
Comment (binodbhusal) [inline README.md]: Please update new ER diagram there is data type mistake in User table created_at_date.
https://drawsql.app/teams/binod-1/diagrams/motorhub-final-capstone-project

File: README.md [hunk 2]
```diff
@@ -1,24 +1,201 @@
...
+# Motor's Reservation <a name="about-project"></a>
 
-* How to run the test suite
+> The Backend for the Motor's Reservation app. Built using Ruby on Rails.
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- ER diagram"-->
 
-* Deployment instructions
+# Database Schema <a name="ER-diagram"></a>
```
Comment (binodbhusal) [inline README.md]: The title of the project is Motorhub. please change it. and Write a **meaningfull description of the project: Like this:** 
The Motorhub - Online Motor Reservation System is a web application designed to provide users with the ability to reserve, view, and manage motors such as cars and bikes reservation for purchases. The system aims to streamline the process of finding and booking motors from various locations, making it convenient for both users and motor providers.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,201 @@
...
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Authentication
+- CRUD
+- Image Upload
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (binodbhusal) [inline README.md]: Replace key features like this : 
1. User Authentication: The system allows users to create accounts and  log in
2. Motors Listings: Motors providers can list their available motors. Each listing includes details such as the brand, model, manufacturer, price, description, and location.
3. Reservation Management: Users can view available motors at different locations and make reservations
4. Add and delete motors in including location

File: README.md [hunk 4]
```diff
@@ -1,24 +1,201 @@
...
+If you liked this project, give it a ⭐️ and kindly send to me an e-mail expressing it, it would make our day and fuel our motivation.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse for this opportunity.
+
```
Comment (binodbhusal) [inline README.md]: update here like this:
Hats off to [Murat Korkmaz](https://www.behance.net/muratk) for designing the awesome template that was used. ✨
Microverse for giving us this chance
The amazing code reviewers for making us improve every day 👍

**Review by binodbhusal — CHANGES_REQUESTED** [review]
Hi @rahimialijan ,
Good job so far! I see you put a effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

Required Changes ♻️
Please Check the comments under the review.
Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: README.md (hunk 1 above)
Comment (rahimialijan) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (rahimialijan) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (rahimialijan) [reply README.md]: I think the CRUD give full details

**Review by dieum512 — APPROVED** [review]
Hey @rahimialijan 

Your project is complete! There is nothing else to say other than... it's time to merge it :ok_hand: 
Congratulations! :tada: 



### HIGHLIGHTS
- [x] No  linter errors :heavy_check_mark: 
- [x] Good Commit messages :heavy_check_mark: 
- [x] Repository documented in a professional way :heavy_check_mark: 
- [x] Made the changes requested by the reviwer :heavy_check_mark:

**Pull request merged**